packages/elements/src/components/ui/buttons/Button.module.css

```diff
@@ -14,7 +14,7 @@
   --swui-button-text-color-focus: var(--swui-button-text-color);
   --swui-button-text-color-hover: var(--swui-button-text-color);
   --swui-button-text-color-active: var(--swui-button-text-color);
-  --swui-button-text-color-disabled: var(--swui-text-disabled-color);
+  --swui-button-text-color-disabled: var(--lhds-color-ui-600);
```

Comment:
Instead of doing this, you should find the definition of `--swui-text-disabled-color` and change its value, like so:
```
--swui-text-disabled-color: var(--lhds-color-ui-600);
```
That way, you only need to make the change in one place :) And if someone wants to override it (say Travel wants disabled text to be orange) they only need to override that variable.

Reply:
👍

Reply:
🎈